pkg/transaction/metric/metricscollector.go

```diff
@@ -161,8 +166,17 @@ func (c *collector) Execute() error {
 	c.usageEndTime = now()
 	c.metricEndTime = now()
 	c.orgGUID = c.getOrgGUID()
-	log.Debugf("Generating usage event [start timestamp: %d, end timestamp: %d]", util.ConvertTimeToMillis(c.usageStartTime), util.ConvertTimeToMillis(c.usageEndTime))
-	log.Debugf("Generating metric event [start timestamp: %d, end timestamp: %d]", util.ConvertTimeToMillis(c.metricStartTime), util.ConvertTimeToMillis(c.metricEndTime))
+	c.logger.
+		WithField("start timestamp", util.ConvertTimeToMillis(c.usageStartTime)).
```

Comment:
I know we have this in other places, but wondering if we should skip the spaces for the field names.  I know json will still be valid, just thinking we may want to make it a standard?